src/app/containers/ATIAnalytics/atiUrl/index.js

```diff
@@ -50,7 +50,9 @@ export const buildATIPageTrackPath = ({
   // For canonical, we have a requirement to encode the x5 and x6 value twice. Source issue: https://github.com/bbc/simorgh/pull/6593
   const x5Value = platform === 'amp' ? href : href && encodeURIComponent(href);
   const x6Value =
-    platform === 'amp' ? referrer : referrer && encodeURIComponent(referrer);
+    platform === 'amp'
+      ? referrer
+      : referrer && encodeURIComponent(encodeURIComponent(referrer));
```

Comment:
Could / should `encodeURIComponent` do the null check, so we don't have to?

e.g. instead of `referrer && encodeURIComponent(referrer)` it would just be `encodeURIComponent(referrer)` which would just return the referrer if it was null? (Just a suggestion, feel free to say no)